### PR TITLE
Initial commit of stanza codebase

### PR DESCRIPTION
**Pull request by djaglowski**
Contribution of observIQ's [stanza](https://github.com/observIQ/stanza) codebase to the OpenTelemetry project, according to the process established [here](https://github.com/open-telemetry/community/issues/605#issuecomment-765621333).

The codebase was very slightly adapted to suit the context of the new project. The notable changes are:
- Add OpenTelemetry copyright notice to all `.go` files
- Rename packages from `github.com/observiq/stanza/*` to `github.com/opentelemetry/opentelemetry-log-collection/*`
- Remove the `MAINTAINERS.md`, `GOVERNANCE.md`, `CONTRIBUTING.md`, since these details are provided through other mechanisms as part of the OpenTelemetry project
- Simplify `README.md`
  - Remove references to the files listed above, since they no longer exist
  - Remove badges, since these will need updating and may be broken initially
  - Add note about project origins and link to deprecated stanza project
- Remove circleCI workflows, at least initially
  - The build/test/release workflow will be needed, but perhaps should be migrated to GitHub Actions
  - The end-to-end performance testing workflow should likely be dropped, since the collector's testbed provides for a simpler benchmarking workflow

### CODE REVIEW

File: LICENSE
Comment (tigrannajaryan) [inline LICENSE:190]: Please use "Copyright The OpenTelemetry Authors" as advised here: https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md#copyright-notices

File: README.md
Comment (tigrannajaryan) [inline README.md:4]:
```suggestion
This project was originally developed by [observIQ](https://observiq.com/) under the name [Stanza](https://github.com/observIQ/stanza). It has been contributed to the OpenTelemetry project in order to accelerate development of the collector's log collection capabilities.
```

File: agent/agent.go
Comment (tigrannajaryan) [inline agent/agent.go:1]: Let's use `Copyright The OpenTelemetry Authors` to avoid the need to update the year.

File: README.md
Comment (tigrannajaryan) [inline README.md:1]: Can we add a notice that the repository is unstable and should not be used outside OpenTelemetry? I want to make sure we are free to make any changes we want and do not break someone who is unaware of what the intent is.

**Review by tigrannajaryan — COMMENTED** [review]
LGTM, with minor comments.
1. For CI it is best to start with github actions since all Otel repos are migrating to it. Can be added in a future PR.
2. make install-tools && make build passes OK.
3. make test fails locally for me, we can address when setting up the CI.
4. logo_small.png is probably not intended to be committed. Not used anywhere.

File: README.md
Comment (djaglowski) [reply README.md:1]: Added a note.

**Comment by tigrannajaryan** [comment]
@djaglowski Looks like the "main" branch was renamed to "master". I am guessing you did it or some other maintainer? If yes any particular reason for that? Note that all of the Otel migrates to "main", so unless we have good reasons to start with "master" we should keep "main" as the branch name.

**Comment by djaglowski** [comment]
@tigrannajaryan, I switched the default to `master`, but only to match the `collector` and `collector-contrib` repos.

I'm happy to handle switching it back, but suggest we do the follow to maintain history, if this seems reasonable:
- Finish processing this PR
- Merge `master` into `main`
- Set `main` as the default branch

**Comment by tigrannajaryan** [comment]
@djaglowski sounds good. Let's switch to main after merging the PR. We are planning to switch Collector repos too.

**Review by bogdandrutu — APPROVED** [review]
From the donation perspective. As discussed we need to setup CI and do other stuff but those are incremental changes

**Pull request merged**
